src/dao/AccountDao.java

```diff
@@ -42,42 +67,27 @@ public AccountDao() {
      * @throws dao.DaoException if there is an error connecting to the database
      * or executing the query
      */
-    public Object getAccount(int accountId) throws DaoException {
-        return null;
-    }
+    public boolean getAccount(int accountId) throws DaoException {
 
-    /**
-     * Saves the specified account by creating a new record or overwriting the
-     * existing account
-     *
-     * @param account the account to save
-     * @throws dao.DaoException if there is an error connecting to the database
-     * or executing the query
-     */
-    public void saveAccount(Object account) throws DaoException {
+        boolean check = false;
+        if (u.Getaccountid() == 001) {
+            check = true;
+
+        }
 
+        return check;
     }
 
-    /**
-     * Permanently deletes the specified account
-     *
-     * @param account the account to delete
-     * @throws DaoException if there is an error connecting to the database or
-     * executing the query
-     */
-    public void deleteAccount(Object account) throws DaoException {
+    public boolean checklogin(String username, String password) {
 
-    }
+        boolean check = false;
 
-    /**
-     * Permanently deletes the specified account
-     *
-     * @param accountId the ID of the account to delete
-     * @throws dao.DaoException if there is an error connecting to the database
-     * or executing the query
-     */
-    public void deleteAccount(int accountId) throws DaoException {
+        if (u.Getusername().equalsIgnoreCase("JDoe")
```

Comment:
Sorry for another change. :( The User u isn't actually instantiated anywhere so it's equal to null. This line will cause a NullPointerException. You could just compare username to JDoe and password to 123456 directly to simplify.

Reply:
No problem. I fixed it.